extensions/sql-migration/src/dialog/validationResults/validateIrDialog.ts

```diff
@@ -405,10 +413,11 @@ export class ValidateIrDialog {
 					return true;
 				}
 			} catch (error) {
```

Comment:
should we log this error in ADS kusto?

Reply:
good catch, I agree all api failures should be logged.